app-frontend/src/app/components/colorCorrectScenes/colorCorrectScenes.controller.js

```diff
@@ -161,4 +162,12 @@ export default class ColorCorrectScenesController {
             this.selectedLayers.delete(scene.id);
         }
     }
+
+    setHoveredScene(scene) {
+        this.onSceneMouseover({ scene: scene });
```

Comment:
I am getting linting errors from this line: `167:31  warning  There should be no space after '{'   object-curly-spacing` and `167:46  warning  There should be no space before '}'  object-curly-spacing`